src/SimpleCalculator.DataAccessLibrary/Model/CalculatorMemory.cs

```diff
@@ -0,0 +1,9 @@
+namespace SimpleCalculator.DataAccessLibrary.Model;
+
+public class CalculatorMemory
```

Comment:
I'd rename it as `CalculationResultEntity` or similar. There is nothing related with memory there